test/simulator/timestamp/src/timestamp_manager.cpp

```diff
@@ -224,65 +224,84 @@ timestamp_manager::validate_read_timestamp(session_simulator *session, const uin
     return (0);
 }
 
-/* Validate the commit timestamp. */
+/*
+ * Validate the commit timestamp. The constraints on the read timestamp are:
```

Comment:
Please, be consistence between `ts` vs `timestamp` and the `_` between `commit/first/oldest` and `timestamp`.